src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1188,4 +1198,11 @@ void RaftReplDev::on_log_found(logstore_seq_num_t lsn, log_buffer buf, void* ctx
 
 void RaftReplDev::on_restart() { m_listener->on_restart(); }
 
+bool RaftReplDev::is_resync_mode() {
+    int64_t const leader_commited_lsn = raft_server()->get_leader_committed_log_idx();
```

Comment:
is `get_leader_committed_log_idx` very light-weight operation (just a read) or it take some mutex while doing it? Asking since this is_resync_mode will be accessed quite frequently in I/O path, so would be good to double confirm.

Reply:
Its a couple of atomics
https://github.com/eBay/NuRaft/blob/0a754e0b95f9763d9f3585973eaf51d0a413f1cd/include/libnuraft/raft_server.hxx#L475

Reply:
it should be fine.